ast.go

```diff
@@ -931,6 +943,12 @@ type CreateRetentionPolicyStatement struct {
 
 	// Shard Duration.
 	ShardGroupDuration time.Duration
+
+	// Error on writes after this duration from now
+	FutureWriteLimit time.Duration
```

Comment:
We set defaults here (as opposed to using pointers with nil designating the default, as in other routines) because we are explicitly creating a retention policy, not merely adding one to a database creation.  So the defaults here are definitive.